ch04.adoc

```diff
@@ -210,304 +226,208 @@ double time(time) ;
+When a datetime is converted to a time coordinate value or vice-versa in this calendar, any leap seconds (positive or negative) must be counted that occurred in the interval between the datetime and the reference datetime in the **`units`**.
+Datetimes in the future are not allowed in this calendar, because it is unknown when future leap seconds will occur.
+A non-zero time zone offset is not allowed in this calendar.
 
-**`tai`**:: A Gregorian calendar __without__ leap seconds that is based on International Atomic Time (TAI).
-Datetimes before 1958-01-01 0:0:0 are not allowed in this calendar.
+**`tai`**:: A Gregorian calendar __without__ leap seconds that is based on International Atomic Time (TAI), which began on 1958-01-01 0:0:0.
+Earlier datetimes than this are not allowed in the **`tai`** calendar.
 For any given instant, the **`tai`** datetime is ahead of the **`utc`** datetime, where "ahead" means the same as it does when describing a timezone to the east as being ahead of one to the west.
-The difference between the two datetimes for a given instant of time is the net number of leap seconds introduced since 1958-01-01.
-The difference was zero on that instant, when both calendars began.
-This means that a given datetime in the **`tai`** calendar represents an instant that is later than the same datetime in the **`utc`** calendar.
-See also <<leap-seconds>>.
+Moreover, a given datetime in the **`tai`** calendar represents an instant that is later than the same datetime in the **`utc`** calendar.
+The difference between the two datetimes (**`tai`** minus **`utc`**) for a given instant of time is the net number of leap seconds introduced since 1958-01-01.
+A non-zero time zone offset is not allowed in this calendar.
 
 **`noleap`** or **`365_day`**:: A calendar with no leap years, i.e., all years are 365 days long, and there are no leap seconds.
 
 **`all_leap`** or **`366_day`**:: A calendar in which every year is a leap year, i.e., all years are 366 days long, and there are no leap seconds.
 
 **`360_day`**:: A calendar in which all years are 360 days, and divided into 30 day months, and there are no leap seconds.
 
-**`none`**:: To be used when there is no annual cycle. See <<none-calendar>>.
+**`none`**:: To be used when there is no annual cycle.
+See <<none-calendar>>.
 
-Any other value may be given to the **`calendar`** attribute to describe an explicitly defined calendar. See <<explicit-calendar>>.
+Any other value may be given to the **`calendar`** attribute to describe an explicitly defined calendar.
+See <<explicit-calendar>>.
 
 
-[[leap-seconds, Section 4.4.3, "Leap Seconds"]]
+[[leap-seconds, Section 4.4.4, "Leap Seconds"]]
 ==== Leap Seconds
 
 This section describes how to deal properly with leap seconds.
```

Comment:
I strongly oppose to any language suggesting that CF has a solution to "how to deal properly with leap seconds". This is an identified  problem that currently is without an accepted solution. There are several alternative technical solutions cf. this presentation https://www.bipm.org/documents/20126/52354616/CCTF_UTC_presentation.pdf/0a8f0954-7c00-9142-8da2-f4221c7e0269 (in particular I draw your attention to viewgraph 12), and this paper https://doi.org/10.1088/1681-7575/ac9da5.